doc/rst/language/spec/procedures.rst

```diff
@@ -120,13 +122,10 @@ Procedures are defined with the following syntax:
      'extern'
      'override'
 
-   function-name:
-     identifier
-     operator-name
-
    operator-name: one of
      + - * / % ** : ! == != <= >= < > << >> & | ^ ~
      = += -= *= /= %= **= &= |= ^= <<= >>= <=> <~>
+     'by' 'align'
```

Comment:
May as well add `#`?

Reply:
I'm experimenting with this a bit.  Not sure why the 'operator' I typed is fine and the 'inline' (etc) designators just above it are fine but these aren't

Reply:
Looks like it's something to do with following the operators, so I'm moving it up before them in the list